docs/cluster-management/configure-queue-service.md

```diff
@@ -0,0 +1,98 @@
+| SECRET_JWT_PRIVATE_KEY | Yes      | A private key uses for signing jwt tokens. Generate one by running `$ openssl genrsa -out jwtqs.pem 2048`                   |
+| SECRET_JWT_PUBLIC_KEY  | Yes      | The public key used for verifying the signature. Generate one by running `$ openssl rsa -in jwtqs.pem -pubout -out jwtqs.pub` |
+| SECRET_JWT_SD_API_PUBLIC_KEY | *none*  | The public key used for verifying the signature of the JWT. Use the same one as configured in the API |
+
+```yaml
+# config/local.yaml
+auth:
+    jwtPrivateKey: |
+        PRIVATE KEY HERE
+    jwtPublicKey: |
+        PUBLIC KEY HERE
+    jwtSDApiPublicKey: |
+        PUBLIC KEY HERE
+```
+
+### Serving
+
+Configure the how the service is listening for traffic.
+
+| Key       | Default             | Description                                                                                                                                                                                                |
+|:----------|:--------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| PORT      | 80                  | Port to listen on                                                                                                                                                                                          |
+| HOST      | 0.0.0.0             | Host to listen on (set to localhost to only accept connections from this machine)                                                                                                                          |
+| URI       | http://localhost:80 | Externally routable URI (usually your load balancer or CNAME)                                                                                                                                              |
+| HTTPD_TLS | false               | SSL support; for SSL, replace `false` with a JSON object that provides the options required by [`tls.createServer`](https://nodejs.org/api/tls.html#tls_tls_createserver_options_secureconnectionlistener) |
+
+```yaml
+# config/local.yaml
+httpd:
+    port: 443
+    host: 0.0.0.0
+    uri: https://localhost
+    tls:
+        key: |
+            PRIVATE KEY HERE
+        cert: |
+            YOUR CERT HERE
+```
+
+### Configure Redis Queue
+
+Configure some settings about storing Build Artifacts.
+
+| Key                | Default              | Description                                         |
+|:-------------------|:---------------------|:----------------------------------------------------|
+| REDIS_HOST    | 127.0.0.1      | Redis host                        |
+| REDIS_PORT    | 9999           | Redis port                        |
```

Comment:
are you missing some vars?
https://github.com/screwdriver-cd/queue-service/blob/master/config/custom-environment-variables.yaml#L216-L220

Reply:
Updated vars